Adjusts Spree::Stock::Estimator Behaviour

DanielWright opened the pull request:
This PR proposes to adjust the estimator's behaviour to evaluate all shipping-methods when the order's source is a POS. The default behaviour remains, only evaluating frontend-only shipping-methods.

braidn commented on app/models/spree/stock/estimator_decorator.rb:
```diff
@@ -0,0 +1,19 @@
+
+      def pos?(package)
+        # Eww.
+        package.shipment.order.pos?
```
.00084 on the RipKarl function-o-meter

Commit:
Improves Shopify Order Importer

The default Solidus stock-estimator will only evaluate shipping-methods
available to the front-end. For retail orders imported from Shopify, we want to
use a hidden, backend-only shipping-method, so a different stock-estimator is
needed to evaluate all shipping-methods for available shipping rates.
